index.html

```diff
@@ -5871,7 +5882,44 @@ <h3><dfn>Get Page Source</dfn></h3>
 </section> <!-- /Get Page Source -->
 
 <section>
-<h3>Executing Script</h3>
+ <h3>Scripts</h3>
+
+ <p>
+  An <dfn>executable script</dfn> is an abstraction used to
+  identify a script when it is transported via the
+  <a href="#protocol">protocol</a>, between <a>remote</a> and
+  <a>local</a> ends.
+
+  Each script has an associated <dfn>executable script name</dfn>
+  that uniquely identifies the script within the <a>current session</a>.
```

Comment:
Right, this makes sense, let me think through the rewrite.

Reply:
So, I was trying to get across the idea that the script itself is the same whether it is pinned or not. Rather than defining a separate "pinned script," it's just a matter of how you obtain it (either from pinned or from the parameter value). I'll change this to `Each script may include an optional associated executable script name` for now and we can address the bigger organizational consideration after I tidy things.

Reply:
How can a script be uniquely identified if it does not have an associated named?

Reply:
It only needs a name if it is pinned, otherwise it is just processed from the provided arguments